README.md

```diff
@@ -51,45 +51,36 @@ Install it
 
     bundle install
 
-### Rails 4.x.x
+Add to your `app/assets/javascripts/application.js` file
 
-Run the generator
-
-    rails generate autocomplete:install
-
-And include autocomplete-rails.js on your layouts
-
-    javascript_include_tag "autocomplete-rails.js"
```

Comment:
Why did you remove this suggestion? I added it to my application.html.haml and it worked.